src/anupSJun20/PalindromeString.java

```diff
@@ -0,0 +1,26 @@
+package anupSJun20;
+/* Program 2:
+Verify given String is palindrome or not. 
+Input: radar
+Output : Given string radar is palindrome.
+
+Input: technocredits
+Output: Given string technocredits is not palindrome.*/
+
+public class PalindromeString {
+	static void findPalindromeString(String str) {
+		String revstr = "";
+		for (int index = str.length() - 1; index >= 0; index--) {
```

Comment:
here we are + operation so String + char will be also fine
Logically this is also good option